image-builder/image_builder.sh

```diff
@@ -39,8 +40,22 @@ Options:
 Extra environment variables:
 	AGENT_BIN:  use it to change the expected agent binary name
 	AGENT_INIT: use kata agent as init process
+	DAX: If 'yes' will build the image with DAX support. The first 2 MB of the
+	     resulting image are reserved for the device namespace information
+	     (metadata) that is used by the guest kernel to enable DAX.
 	USE_DOCKER: If set will build image in a Docker Container (requries docker)
 	            DEFAULT: not set
+
+
+	When DAX is 'yes', the following diagram shows how a 128M image will looks like:
+		 .-----------------------------------.
+		 |-- 2 MB --|-------- 126 MB --------|
+		 | Metadata | Rootfs (/bin,/usr,etc) |
+		 '-----------------------------------'
+
+		 The resulting image can be mounted if the offset of 2 MB is specified:
+		 $ sudo losetup -v -fP -o $((2*1024*1024)) kata-containers.img
```

Comment:
Is there a way for end users a way to verify the image has dax? or not ?  I know some people open the image directly to modify as needed and do some testing in Kata images.